index.html

```diff
@@ -84,7 +83,163 @@ <h1 class="primary py-2 my-2">
       </div>
     </section>
 
-    <
+    <section class="grid text-center m-4 p-2 projects">
+      <h1 class="primary py-2 my-4 text-center">My Recent Work</h1>
+
+      <div class="card my-2">
+        <div class="card-body d-flex flex-column justify-content-center">
+          <p class="placeholder-glow">
+            <span
+              class="placeholder col-12 bg-primary project-sample-img"
+            ></span>
+          </p>
+
+          <h5 class="card-title">Datafy</h5>
+          <ul>
+            <li class="badge text-bg-secondary">REACT</li>
+            <li class="badge text-bg-secondary">Big Query</li>
+            <li class="badge text-bg-secondary">Shopify API</li>
+            <li class="badge text-bg-secondary">
+              Google Cloud Platform Services
+            </li>
+            <li class="badge text-bg-secondary">ExpressJS</li>
+            <li class="badge text-bg-secondary">MySQL</li>
+          </ul>
+          <p class="card-text">
+            App that interacts with the Shopify API to import statistics to a
+            series of Google Cloud Platform services, to analyze the data and
+            present it to the user using a REACT frontend.
+          </p>
```

Comment:
- [ ] KIndly make sure to remove the project description in order to meet with the provided Figma design because on Figma design we don't have that description.

   - ### YOUR DESIGN
   ![image](https://user-images.githubusercontent.com/87197412/200127096-1f48d095-830a-4d36-8120-903d4b6dd09a.png)

  - ### FIGMA DESIGN
   ![image](https://user-images.githubusercontent.com/87197412/200127106-a362efce-1127-4446-9480-634f99add237.png)

Reply:
Fixed